src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,18 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery extends Ball {
+    private static final int MAX_NUMBER = 101;

```

Comment:
```suggestion
    private static final int MAX_NUMBER = 100;
```
magic numbers explains what the number means. The task description required max number 100 for a ball, so it's not clear why you have max number 101. To achieve your goal, you need to make sure your 12th line returns 100 as maximum. So it's getRandomBalls()'s work to handle nextInt returns 100 exclusively.